simpeg_drivers-assets/uijson/tdem1d_inversion.ui.json

```diff
@@ -0,0 +1,581 @@
+        "value": 25,
+        "enabled": true,
+        "verbose": 2
+    },
+    "starting_chi_factor": {
+        "group": "Sparse/blocky model",
+        "label": "IRLS start chi factor",
+        "enabled": true,
+        "value": 1.0,
+        "tooltip": "This chi factor will be used to determine the misfit threshold after which IRLS iterations begin.",
+        "verbose": 3
+    },
+    "beta_tol": {
+        "group": "Update IRLS directive",
+        "label": "Beta tolerance",
+        "value": 0.5,
+        "min": 0.0001,
+        "verbose": 3,
+        "visible": false
+    },
+    "prctile": {
+        "group": "Update IRLS directive",
+        "label": "Percentile",
+        "value": 95,
+        "max": 100,
+        "min": 5,
+        "verbose": 3,
+        "visible": false
+    },
+    "chi_factor": {
+        "min": 0.1,
+        "max": 20.0,
+        "precision": 1,
+        "lineEdit": false,
+        "group": "Cooling schedule/target",
+        "label": "Chi factor",
+        "value": 1.0,
+        "enabled": true,
+        "tooltip": "The global target data misfit value."
+    },
+    "auto_scale_misfits": {
+        "group": "Cooling schedule/target",
+        "label": "Auto-scale misfits",
+        "value": true,
+        "verbose": 3,
+        "visible": true,
+        "tooltip": "Whether to auto-scale misfits functions (tile, frequency, joint methods) based on chi-factor."
+    },
+    "initial_beta_ratio": {
+        "min": 0.0,
+        "precision": 2,
+        "group": "Cooling schedule/target",
+        "optional": true,
+        "enabled": true,
+        "label": "Initial beta ratio",
+        "value": 100.0,
+        "verbose": 2,
+        "tooltip": "Estimate the trade-off parameter by scaling the ratio between the largest derivatives in the objective function gradients."
+    },
+    "initial_beta": {
+        "min": 0.0,
+        "group": "Cooling schedule/target",
+        "optional": true,
+        "enabled": false,
+        "dependency": "initial_beta_ratio",
+        "dependencyType": "disabled",
+        "label": "Initial beta",
+        "value": 1.0,
+        "verbose": 2,
+        "tooltip": "Trade-off parameter between data misfit and regularization."
+    },
+    "coolingFactor": {
+        "group": "Cooling schedule/target",
+        "label": "Beta cooling factor",
+        "tooltip": "Each beta cooling step will be calculated by dividing the current beta by this factor.",
+        "value": 2.0,
+        "min": 1.1,
+        "max": 100,
+        "precision": 1,
+        "lineEdit": false,
+        "verbose": 2
+    },
+    "coolingRate": {
+        "group": "Optimization",
+        "label": "Iterations per beta",
+        "value": 2,
+        "min": 1,
+        "LineEdit": false,
+        "max": 10,
+        "precision": 1,
+        "verbose": 2,
+        "enabled": true,
+        "tooltip": "Set the number of iterations per beta value. Use higher values for more non-linear optimization problems."
+    },
```

Comment:
```suggestion
    },
    "epsilon_cooling_factor": 1.2,
```

Reply:
Why is that?

Reply:
it's to prevent the input file warnings since epsilon_cooling_factor is an attribute of the inversion options classes.

Reply:
I pushed some other changes to this effect.